geomstats/_backend/tensorflow/__init__.py

```diff
@@ -14,7 +14,7 @@
     asin as arcsin,
     atan2 as arctan2,
     clip_by_value as clip,
-    concat as concatenate,
+    concat,
```

Comment:
Do you want to just delete it maybe?

Reply:
No, I just needed to pass the default `axis=0` to match numpy's behaviour